dev-clean/headers/llsc/cli.hpp

```diff
@@ -1,6 +1,11 @@
 #ifndef LLSC_CLI_HEADERS
 #define LLSC_CLI_HEADERS
 
+#include "filesys.hpp" // for initial_fs, default_sym_file_size
```

Comment:
should not be included here, `llsc.hpp` already includes everything, reorder how `llsc.hpp` includes things if necessary.